source/administration/changelog.md

```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.1
+
+Release date: 2018-07-16
+
+### Highlights
+
+#### Gfycat integration
+ - Added easy access to sharing GIFs without leaving the Mattermost interface.
+ 
+#### Auto-linking plugin (Beta)
+ - Certain messages can now be formatted into Markdown links automatically before they are saved to the Mattermost database.
+
+#### Support Mattermost on a subpath
+ - Added support for hosting Mattermost at any route (e.g., https://www.example.com/mm) with newly added subpath support.
+ 
+#### CSV Compliance Export
+ - Removed existing Compliance Reports feature and replaced it with an improved Compliance Export feature that captures more data in CSV format.
+
+### Improvements
+
+#### Web User Interface
+ - Added member count for the direct message list.
```

Comment:
Which list?

Reply:
https://mattermost.atlassian.net/browse/MM-5761

Reply:
Looks like that ticket is open and I don't see a count in the direct messages "more" menu, so I'll remove it